mobile/src/modules/home/components/HomeStackedAreaChart.tsx

```diff
@@ -33,23 +33,16 @@ interface State {
   selectedDateRange: number;
 }
 
-const styles = {
+const styles = StyleSheet.create({
   stackedAreaChart: { height: 200, paddingVertical: 16, zIndex: 1 },
   yAxis: { position: 'absolute', top: 0, left: 5, height: 200, paddingVertical: 16, zIndex: 2 },
-  svg: {
-    fontSize: 8,
-    fill: 'white', // StyleSheet.create doesn't like fill
-    stroke: 'black',
-    strokeWidth: 0.1,
-    alignmentBaseline: 'baseline',
-    baselineShift: '3'
-  },
   button: {
     position: 'absolute',
-    top: 20,
+    top: 142,
```

Comment:
This is probably not going to work on some devices/orientations, will have to find a better way soon.